roles/ethereum_node/tasks/validations.yaml

```diff
@@ -13,3 +13,10 @@
     success_msg: "{{ ethereum_node_cl }} is a supported client"
     fail_msg: >
       ethereum_node_el = '{{ ethereum_node_cl }}' is invalid. You can choose between: '{{ ethereum_node_supported_cl }}'
+
+- name: Check if consensus client supports blobber configuration
+  ansible.builtin.assert:
+    that:
+      - ethereum_node_blobber_enabled != true or (ethereum_node_cl != 'nimbus' and ethereum_node_cl != 'teku')
```

Comment:
Not sure if it's just me, but my brain got wonky reading this condition. Maybe we could use the `ansible.builtin.fail` here and do something like:

```yaml
ansible.builtin.fail:
  msg: "Error: The blobber tool currently doesn't support nimbus or teku" 
when: ethereum_node_blobber_enabled and ( (ethereum_node_cl != 'nimbus' or ethereum_node_cl != 'teku')
  ```

Reply:
accepted suggestion :D